_utilities/display.md

```diff
@@ -265,7 +265,7 @@ utilities:
 
     <section class="utility-examples">
 
-      <p class="utility-note"><strong>Note:</strong> <a href="http://www.w3.org/TR/css3-box/#overflow-x">Per the W3C overflow spec:</a> The computed values of ‘overflow-x’ and ‘overflow-y’ are the same as their specified values, except that some combinations with ‘visible’ are not possible: if one is specified as ‘visible’ and the other is ‘scroll’ or ‘auto’, then ‘visible’ is set to ‘auto’. The computed value of ‘overflow’ is equal to the computed value of ‘overflow-x’ if ‘overflow-y’ is the same; otherwise it is the pair of computed values of ‘overflow-x’ and ‘overflow-y’.<br/><br/>Any element that includes scrollable content must also be selectable with the keyboard. Add a <code>tabindex="0"</code> property to elements that include scrollable content.</p>
```

Comment:
Could not find the link or content from this reference to W3C. Should we continue to include?

Reply:
@amyleadem I think this is the new link https://www.w3.org/TR/css-overflow-3/

Reply:
Thanks, @mejiaj. Updated in [72ac74b](https://github.com/uswds/uswds-site/pull/1735/commits/72ac74b424366b37775ab819d8faace993026f68)